From 006f08373cbaa8485311f491ab8e67433c0e9ad6 Mon Sep 17 00:00:00 2001
From: Daniel Mack <daniel@zonque.org>
Date: Tue, 24 Oct 2017 15:46:19 +0200
Subject: [PATCH] tests: make accessibility tests conditional

Only test the accessibility functions if accessibility was enabled in the Qt
config.
---
 tests/auto/widgets/widgets.pro | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tests/auto/widgets/widgets.pro b/tests/auto/widgets/widgets.pro
index 441eea0f..7bab18f0 100644
--- a/tests/auto/widgets/widgets.pro
+++ b/tests/auto/widgets/widgets.pro
@@ -1,7 +1,6 @@
 TEMPLATE = subdirs
 
 SUBDIRS += \
-    qwebengineaccessibility \
     qwebenginedefaultsurfaceformat \
     qwebenginedownloads \
     qwebenginefaviconmanager \
@@ -14,6 +13,10 @@ SUBDIRS += \
     qwebenginesettings \
     qwebengineview
 
+qtConfig(accessibility) {
+    SUBDIRS += qwebengineaccessibility
+}
+
 contains(WEBENGINE_CONFIG, use_spellchecker):!cross_compile {
     !contains(WEBENGINE_CONFIG, use_native_spellchecker) {
         SUBDIRS += qwebenginespellcheck
-- 
2.13.6

